### DISCUSSION TITLE
Storybook structure forms and components

### DISCUSSION THREAD

**Discussion by b1aserlu** [swisspost/design-system#2209] [Polls]
original issue Group Form Elements #976

#2172 beside components: https://preview-2172--swisspost-design-system-next.netlify.app/
#2175  as a subfolder in components: https://preview-2175--swisspost-design-system-next.netlify.app/
![image](https://github.com/swisspost/design-system/assets/141234680/c291ad09-d82c-4f16-b0f9-d894bd000175)
![image](https://github.com/swisspost/design-system/assets/141234680/afc66648-7978-4155-ab18-fac2784cbbb5)

**Comment by imagoiq** [comment]
As a subfolder in components + alphabetically ordered

**Reply by b1aserlu** [reply]
added alphabetical order to the preview.ts for all stories.

note: This will sort all the stories by alphabetical order that are not specifically mentioned in the order object.

https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy

**Comment by gfellerph** [comment]
Discussion from the 23.11. 
If form components are moved to a folder under components, the header docs and the icons can go there as well.

**Comment by gfellerph** [comment]
Closing as implemented
